js/src/wp-seo-term-scraper.js

```diff
@@ -280,6 +282,26 @@
 	}
 
 	/**
+	 * Create a term specific Assessor object
+	 * @param {object} i18n The i18n object to use in the assessor.
+	 * @returns {SEOAssessor} The modified Assessor object.
+	 */
+	function createTermScraperAssessor( i18n ) {
+		var assessor = new SEOAssessor( i18n );
+
+		assessor.removeAssessment( 'textLength' );
```

Comment:
Wouldn't it be better if we tell the SEOAssessor what kind of assessments we want, or even have two different assessors if the assessments differ this much?